crypto/o_time.c

```diff
@@ -41,6 +41,10 @@ struct tm *OPENSSL_gmtime(const time_t *timer, struct tm *result)
     if (gmtime_r(timer, result) == NULL)
         return NULL;
     ts = result;
+#elif defined (OPENSSL_SYS_WINDOWS) && defined(_MSC_VER) && _MSC_VER >= 1700
```

Comment:
MSDN suggest that it goes as deep as VS2008 and I can confirm that. So that one can drop to 1500.

Reply:
One can even go to 1400...

Reply:
Thanks for verification. I'm used to not trust MSDN and didn't care enough to go that far into history :) Will change to 1400